activesupport/lib/active_support/core_ext/object/to_query.rb

```diff
@@ -16,6 +16,11 @@ def to_query(key)
 end
 
 class NilClass
+  # Returns a CGI-escaped +key+.
+  def to_query(key)
+    "#{CGI.escape(key.to_param)}"
```

Comment:
What is the significance of the wrapping String interpolation?

Can `CGI.escape` return something other than a String? Does interpolation produce something different from `to_s`?

Reply:
Nah you're right.